recipe/meta.yaml

```
@@ -11,22 +11,23 @@ source:
 
 build:
   noarch: python
-  number: 0
+  number: 1
   script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv"
 
 requirements:
   host:
-    - python
+    - python {{ python_min }}
     - pip
     - setuptools
```

https://dev.azure.com/conda-forge/feedstock-builds/_build/results?buildId=1247519&view=logs&j=7b6f2c87-f3a7-5133-8d84-7c03a75d9dfc&t=9eb77fd2-8ddd-5444-8fc0-71cb28dcb736&l=895
> Traceback (most recent call last):
>     File "<string>", line 2, in <module>
>     File "<pip-setuptools-caller>", line 35, in <module>
>     File "/home/conda/feedstock_root/build_artifacts/scikit-video_1747333303889/work/setup.py", line 40, in <module>
>       from numpy.distutils.core import setup
>   ModuleNotFoundError: No module named 'numpy'

```suggestion
    - setuptools
    - numpy
    - scipy
```